test/features/support/db_read.js

```diff
@@ -6,168 +6,180 @@ let specDatabaseRead;
+// Scenario: The user is not able to receive any Digital Registries database information with schema version because of not providing route param
 When(
-  'The user triggers an action without a route param to display one database schema versions',
+  'The user triggers an action without a route param to display one database schema version',
   () => {
     specDatabaseRead
       .get(baseUrl)
       .withHeaders(`${header.key}`, `${header.value}`);
   }
 );
 
-Then('The result of an operation to receive any Digital Registries database information with schema versions returns an invalid route param error', async () => {
-  await specDatabaseRead.toss();
-  specDatabaseRead.response().should.have.status(400);
-  specDatabaseRead.response().should.have.body('{"Invalid route param id"}');
-});
-
+Then(
+  'The result of an operation to receive any Digital Registries database information with schema version returns an invalid route param error',
+  async () => {
+    await specDatabaseRead.toss();
+    specDatabaseRead.response().should.have.status(400);
+    specDatabaseRead.response().should.have.body('{"Invalid route param id"}');
+  }
+);
 
-// Scenario: The user is not able to receive any Digital Registries database information with schema versions because of empty value in the header
+// Scenario: The user is not able to receive any Digital Registries database information with the schema version because of an empty value in the header
 When(
-  'The user triggers an action with an invalid payload to display one database schema versions', () => {
+  'The user triggers an action with an invalid payload to display one database schema version',
+  () => {
+    specDatabaseRead.get(baseUrl).withHeaders(`${header.key}`, '');
+  }
+);
+
+Then(
+  'The result of an operation to receive any Digital Registries database information with schema version returns an error',
+  async () => {
+    await specDatabaseRead.toss();
+    specDatabaseRead.response().should.have.status(400);
     specDatabaseRead
-    .get(baseUrl)
-    .withHeaders(`${header.key}`, '');
+      .response()
+      .should.have.body(
+        '{"Invalid format of Information-Mediator-Client, should match INSTANCE/CLASS/MEMBER/SUBSYSTEM"}'
+      );
   }
 );
-  
-Then('The result of an operation to receive any Digital Registries database information with schema versions returns an error', async () => {
-  await specDatabaseRead.toss();
-  specDatabaseRead.response().should.have.status(400);
-  specDatabaseRead.response().should.have.body('{"Invalid format of Information-Mediator-Client, should match INSTANCE/CLASS/MEMBER/SUBSYSTEM"}');
-});
 
-// Scenario: The user is not able to receive any Digital Registries database information with schema versions because of not including header
+// Scenario: The user is not able to receive any Digital Registries database information with the schema version because of not include a header
```

Comment:
Maybe:  ```The user is not able to receive database information from Digital Registries with the schema version because there is no header included```